lib/index.js

```diff
@@ -107,6 +103,19 @@ function createModel(options) {
     }
 }
 
+/**
+ * Create a middleware chain that executes a specific conductor
+ * @public
+ * @function createConductorHandlers
+ * @param {Object} conductor a Conductor instance
+ * @returns {Function[]}
+ */
+function createConductorHandlers(conductor) {
+    return [
```

Comment:
Can we add an assertion here, like on https://github.com/restify/conductor/pull/121/files#diff-6d186b954a58d5bb740f73d84fe39073R47?